src/processing/script/transformers/index.js

```diff
@@ -10,6 +10,7 @@ import documentWriteTransformer from './document-write';
 import evalTransformer from './eval';
 import evalCallApplyTransformer from './eval-call-apply';
 import getEvalTransformer from './eval-get';
+import getWindowEvalTransformer from './window-eval-get';
```

Comment:
It's `evalGetTransformer` and `windowEvalGetTransformer`. The word order is the same as in the filename